runner/src/run.rs

```diff
@@ -85,6 +99,11 @@ fn launch_qemu(args: &RunArgs, miralis: PathBuf, firmware: PathBuf) -> ExitCode
     if let Some(cpu) = &cfg.qemu.cpu {
         qemu_cmd.arg("-cpu").arg(cpu);
     }
+
+    if is_qemu_version_6() {
+        qemu_cmd.arg("-cpu").arg("-cpu rv64,h=true");
```

Comment:
Isn't this supposed to be `qemu_cmd.arg("-cpu").arg("rv64,h=true");` ? You repeated `-cpu` twice.